### ISSUE TITLE
Episode #6

### ISSUE THREAD

**Issue by ShalokShalom** [thedevaspect/aspect-fsharp#1]
Hi Dave, I guess you are a little bit busy in your job. 

I currently intelligible your videos and in Episode 5, you say that with computation expressions, parsing gets a lot easier. Can you recommend me a link?

And do think Episode 6 will appear soon?

**Comment by ShalokShalom** [comment]
This might help you: http://gusty.github.io/FSharpPlus/computation-expressions.html

**Comment by frankhale** [comment]
Yeah I loved the videos but was sad to find out that the 6th video was never made =(

**Comment by ShalokShalom** [comment]
He once told me on Twitter, one will soon follow. While that was a few months ago. So so much, this happens.
